MapleServer2/Commands/Game/ItemCommands.cs

```diff
@@ -0,0 +1,50 @@
+﻿using MapleServer2.Commands.Core;
+using MapleServer2.Tools;
+using MapleServer2.Enums;
+using MapleServer2.Data.Static;
+using MapleServer2.Types;
+using System;
+
+namespace MapleServer2.Commands.Game
+{
+    public class ItemCommand : InGameCommand
+    {
+        public ItemCommand()
+        {
+            Aliases = new[]
+            {
+                "item"
+            };
+            Description = "Give an item to the current player.";
+            AddParameter<int>("id", "Item id");
+            AddParameter<int>("amount", "Amount of the same item.");
+            AddParameter<int>("rarity", "Item rarity.");
+        }
+
+        public override void Execute(GameCommandTrigger trigger)
+        {
+            int itemId = trigger.Get<int>("id");
+            int rarity = trigger.Get<int>("rarity") != 0 ? trigger.Get<int>("rarity") : 1;
```

Comment:
What if rarity is 0, we use the default rarity from the item instead of 1?
And can' you set the default value for amount as 1?